app/actions/favorites.js

```diff
@@ -56,7 +58,12 @@ export function addFavoriteAlbum(album) {
 
 export function removeFavoriteAlbum(album) {
   const favorites = store.get('favorites');
-  _.remove(favorites.albums, { id: album.id });
+  if (album.uuid) {
```

Comment:
I've used the `uuid` but I'm not sure about it. I've added handling of both `id` and `uuid` for backward compatibility.